app/database/user.go

```diff
@@ -54,3 +56,54 @@ func (u *User) GetManagedGroupsWithCanGrantGroupAccessIds(store *DataStore) []in
 
 	return managedGroupsWithCanGrantGroupAccessIds
 }
+
+// CanWatchMembersOnParticipant checks whether the user can_watch_members on a participant
+func (u *User) CanWatchMembersOnParticipant(s *DataStore, participantID int64) bool {
+	userCanWatchMembersOnParticipant, err := s.ActiveGroupAncestors().ManagedByUser(u).
+		Where("groups_ancestors_active.child_group_id = ?", participantID).
+		Where("group_managers.can_watch_members").
+		Select("1").
+		Limit(1).
+		HasRows()
+	mustNotBeError(err)
+
+	return userCanWatchMembersOnParticipant
+}
+
+// For Next PR
+// CanRequestHelpTo checks whether the user can request help on an item to a group.
+// func (u *User) CanRequestHelpTo(s *DataStore, itemID, helperGroupID int64) bool {
+//	// in order to verify that the user “can request help to” a group on an item we need to verify whether
+//	// one of the ancestors (including himself) of User has the can_request_help_to(Group) on Item,
+//	// recursively on Item’s ancestors while request_help_propagation=1, for each Group being a descendant of Group.
```

Comment:
such code shouldn't be committed... or when do I review it, in next PR? .. ok you can leave it if it is easier for you.

Reply:
haha, yes it was just to make the process of chunking the PR easier